samples/exampleVerifiedTickets.csv

```diff
@@ -0,0 +1,4 @@
+placed,all_guests,
```

Comment:
We need to sort out what this VerifiedTickets.csv is about. I know that sounds vague. I pulled the column headers from the code, seems like we were using it to compare for ticket owner changes but I don't recall where this spreadsheet was originating. Someone was maybe manually creating them and sending to me(?) 
More generally need to review how reports work in the context of ticket owner/guest changes

Reply:
@mpesavento I going to merge this PR but this is a thing we prolly should investigate